packages/loader/container-loader/src/utils.ts

```diff
@@ -24,12 +25,11 @@ export function parseUrl(url: string): IParsedUrl | undefined {
     if (typeof parsed.pathname !== "string") {
         throw new Error("Failed to parse pathname");
     }
-
+    const query = parsed.search ?? "";
```

Comment:
For a url such as "http://localhost/default?query=param", what value is assigned to query?
Is it - "query=param" or "?query=param". Basically, does it start with question mark.

Reply:
looks like it does